src/tools/stereoCalib/include/stereoCalibThread.h

```diff
@@ -84,30 +86,38 @@ class stereoCalibThread : public Thread
     string boardType;
     char pathL[256];
     char pathR[256];
+	
+	string calibFileName;
+
+	string cfiFolder;
+	string cfiCalibFile;
+
```

Comment:
As @traversaro already pointed out, it seems the indentation is somehow messed up here and there.